src/integrated/soil_snow_model.jl

```diff
@@ -387,22 +387,19 @@ function soil_boundary_fluxes!(
     bc = soil.boundary_conditions.top
     p.soil.turbulent_fluxes .= turbulent_fluxes(bc.atmos, soil, Y, p, t)
     p.soil.R_n .= net_radiation(bc.radiation, soil, Y, p, t)
-    # influx = maximum possible rate of infiltration given precip, snowmelt, evaporation/condensation
-    # but if this exceeds infiltration capacity of the soil, runoff will
-    # be generated.
-    # Use top_bc.water as temporary storage to avoid allocation
-    influx = p.soil.top_bc.water
-    @. influx =
-        p.drivers.P_liq +
-        p.snow.water_runoff * p.snow.snow_cover_fraction +
-        p.excess_water_flux +

```

Comment:
we need to keep these changes.

Reply:
this was an artifact of the older state of my branch during the update, i believe

Reply:
artifact from rebasing i believe, this has been rectified.